docs/spark/tutorials/get-started-macos-instructions.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Get Started with .NET for Apache Spark on MacOSX
+description: Discover how to run a .NET for Apache Spark app using .NET Core on MacOSX.
+ms.date: 12/30/2019
+ms.topic: tutorial
+ms.custom: mvc
+# Customer intent: As a developer, I want to write a simple custom application using .NET for Apache Spark on MacOSX.
+---
+
+# Tutorial: Get Started with .NET for Apache Spark on MacOS
+
+This tutorial will show you how to run a .NET for Apache Spark app using .NET Core on MacOSX.
+
+> [!div class="checklist"]
+>
+> * Pre-requisites to run your .NET for Apache Spark application
+> * Write a simple .NET for Apache Spark application
+> * Build and run your .NET for Apache Spark application
+
+## Pre-requisites
+
+- Download and install **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** 
+- Install **[Java 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)** 
+  - Select the appropriate version for your operating system e.g., `jdk-8u231-macosx-x64.dmg`.
+  - Install using the installer and verify you are able to run `java` from your command-line
+- Download and install **[Apache Spark 2.4.4](https://archive.apache.org/dist/spark/spark-2.4.4/spark-2.4.4-bin-hadoop2.7.tgz)**:
+  - Add the necessary environment variables SPARK_HOME e.g., `~/bin/spark-2.4.4-bin-hadoop2.7/`
+  
+    ```bash
+    export SPARK_HOME=~/bin/spark-2.4.4-bin-hadoop2.7/
+    export PATH="$SPARK_HOME/bin:$PATH"
+    source ~/.bashrc
+    ```
+    
+- Download and install **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release:
+  - Select a **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release from .NET for Apache Spark GitHub Releases page and download into your local machine (e.g., `/bin/Microsoft.Spark.Worker/`).
+  - **IMPORTANT** Create a new environment variable using ```export DOTNET_WORKER_DIR <your_path>``` and set it to the directory where you downloaded and extracted the Microsoft.Spark.Worker (e.g., `/bin/Microsoft.Spark.Worker/`).
+
+## Authoring a .NET for Apache Spark App
+
+- Use the `dotnet` CLI to create a console application.
+
+    ```
+    dotnet new console -o HelloSpark
+    ```
+    
+- Install `Microsoft.Spark` Nuget package into the project from the [spark nuget.org feed](https://www.nuget.org/profiles/spark) - see [Ways to install Nuget Package](https://docs.microsoft.com/en-us/nuget/consume-packages/ways-to-install-a-package)
+    
+    ```
+    cd HelloSpark
+    dotnet add package Microsoft.Spark
+    ```
```

Comment:
@mairaw, this needs an identifier. Currently, some snippets with mixed commands like this has dotnetcli, and others has console. (#16462)